index.js

```diff
@@ -176,8 +176,14 @@ Client.prototype.validateXmlrpcVersion = function( callback ) {
 			return callback( error );
 		}
 
-		if ( xmlrpcVersion !== version ) {
-			return callback( new Error( "Mismatching versions for Gilded WordPress. " +
+		// The server should have all capabilities expected by this client.
+		// The server must therefore be in the "^x.y.z" semver-range, whereby it
+		// implements the same major version, and the same (or newer) minor version.
+		var xmlrpcVersionParts = xmlrpcVersion.split( "." );
+		var clientVersionParts = version.split( "." );
+
+		if ( !( xmlrpcVersionParts[0] === clientVersionParts[0] && xmlrpcVersionParts[1] >= clientVersionParts[1] ) ) {
```

Comment:
String comparisons will fail at double digits, e.g., `"10" > "2" // false`.

I'm fine with this simple check if you want to update it, but I'm not opposed to pulling in `semver` to do the check.

Reply:
I went back and forth between using parseFloat and string split. I realise now I should have combined the two. We need split to ensure 2.20 doesn't become 2.2, and we need parseInt to make sure 20 is a number and compared as such.